stripe/src/main/java/com/stripe/android/paymentsheet/PaymentSheet.kt

```diff
@@ -3,7 +3,6 @@ package com.stripe.android.paymentsheet
 import android.os.Parcelable
 import androidx.activity.ComponentActivity
 import androidx.fragment.app.Fragment
```

Comment:
We use the word "Flow" in a bunch of different contexts, and it seems like this particular "Flow" really relates to the PaymentOptions?  Would it make sense to replace the DefaultFlow..., FlowFactory..., FlowFactoryInit... with something that more strongly identifies it with PaymentOptions or the custom PaymentSheet?

Reply:
For a little more details here are some ways we use the word flow in our SDK:

- DefaultFlowController
- PaymentFlowActivity
- DefaultFlowResultProcessor
- DefaultPaymentFlowResultProcessor
- PaymentFlowResult 
- Kotlin flows!

Reply:
I'll do this in a separate PR.

Reply:
That's a good idea